glossary/repair.md

```diff
@@ -0,0 +1,8 @@
+type of [achievable](https://github.com/newatoms/guides/blob/ready/glossary/achievable.md), where you repair something which was broken, i.e. not working the way it should.
+
+*Repairs* are documented by simply stating what is broken and, if applicable, how to replicate the broken thing. 
+
+Examples: 
+The log out button gives an error in the Interface. If you open the web inspector while in the Interface and click on the button log out in the navigation menu, an error is displayed.
```

Comment:
"click on the button log out in the navigation menu" is unclear. I think "log out" should be in quotes each time it's used.